tests/integration/gh-6970/.dockerignore

```diff
@@ -0,0 +1 @@
+target/*
```

Comment:
There should not be a `.dockerignore` file, if this is not a docker based test

Reply:
Removed